src/gsy_e/models/strategy/energy_parameters/energy_params_eb.py

```diff
@@ -77,31 +243,70 @@ def event_traded_energy(
         for time_slot, energy_value_kWh in trade_profile.items():
             self._state.decrement_energy_requirement(
                 purchased_energy_Wh=energy_value_kWh * 1000,
-                time_slot=time_slot,
+                time_slot=time_slot.set(tz=pendulum.UTC),
```

Comment:
This alarms me. IMO this should ne already done in `generate_trade_profile` or more specific in `_expand_profile_slots`. What do you think? 
Of course, this can be done in a separate PR.

Reply:
I think that the code works even without the timezone assignment, let me give it a try by removing all these. I will also adapt your suggestion to strip the timezone information from the point that this profile is generated. Thanks!

Reply:
pendulum's default timezone is already UTC when creating new DateTime objects:
```py
>>> pendulum.datetime(2022, 1, 1)
DateTime(2022, 1, 1, 0, 0, 0, tzinfo=Timezone('UTC'))
```
so (in theory) we should be already fine unless some part of the code explicitly sets this to something different, or in cases where we don't use pendulum objects.

Reply:
We decided to always explicitly set the time zone because I remember having issues with running tests locally when the timezone is set differently on you computer. Also, in order to change it globally if it needs to be change in the future.